win/nuget/c-ares/Roaster.c-ares.v142.dyn.x64.nuspec

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0"?>
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
   <metadata>
-    <id>Roaster.c-ares.v141.dyn.x64</id>
+    <id>Roaster.c-ares.v142.dyn.x64</id>
```

Comment:
Please move all 141->142 change into a dedicated PR.
These will be breaking change, I'll also advance Roaster version tag for it once merged.

Reply:
Sure. This makes sense.